.changeset/great-grapes-flow.md

```diff
@@ -0,0 +1,5 @@
+---
+"@chakra-ui/transition": major
```

Comment:
Kindly change this to a `minor`. I don't see it as a breaking change since it'll work as it used to, you're only making it possible to pass custom variants